zebra/redistribute.c

```diff
@@ -548,6 +548,7 @@ void zebra_interface_address_add_update(struct interface *ifp,
 						client, ifp, ifc);
 		}
 	}
+	zebra_interface_nhg_reinstall(ifp);
```

Comment:
could we just have a comment here, saying what we're doing?
like "we may have nhgs that were implicitly deleted, and we need to bring zebra and the dataplane back into sync"